lib/crypto.js

```diff
@@ -52,6 +52,10 @@ const {
   pbkdf2,
   pbkdf2Sync
 } = require('internal/crypto/pbkdf2');
+const {
+  scrypt,
+  scryptSync
+} = require('internal/crypto/scrypt');
```

Comment:
It would be great to actually lazy load this as it is definitely not always required when loading crypto.

Reply:
You could say the same thing for pbkdf2 and others.  Comment noted but I'd like to save that for another PR.